apis/nucleus/src/utils/cell-padding.js

```diff
@@ -41,7 +41,8 @@ const getPadding = ({ layout, isError, senseTheme, titleStyles }) => {
   if (isCardTheme) {
     const showTitle = layout?.showTitles && !!layout?.title;
     const showSubtitle = layout?.showTitles && !!layout?.subtitle;
-    const showFootnote = layout?.showTitles && !!layout?.footnote;
+    const hasFilters = layout?.filters?.length > 0 && layout?.qHyperCube?.qMeasureInfo?.length > 0;
+    const showFootnote = layout?.showTitles && (!!layout?.footnote || hasFilters);
```

Comment:
also need to do `const filtersFootnoteString = generateFiltersString(layout?.filters ?? [], translator);` and check if filtersFootnoteString is empty string or not